### ISSUE TITLE
No fallback solution for Elements if Matrix.org is unavailable.

### ISSUE THREAD

**Issue by OrvilleQ** [vector-im/element-web#25079]
### Steps to reproduce

1. Ban matrix.org.
2. Open element desktop
3. Ooops

#### What did you expect?
I was trying to share my Matrix server with friends in China, Matrix.org is blocked by gfw, and they cannot use element to connect to my server because if matrix.org is unavailable the application won't start in the first place and has no fallback solution.

Same thing happened to Android client too, I have no apple device but I believe this might be the same on iOS.

#### What happened instead?
It should at least provide a easy to use fallback solution for non tech user to select server they gonna use.

### Will you send logs?

No

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/element-web/issues/11154
